Project.toml

```diff
@@ -0,0 +1,9 @@
+name = "JavaCall"
+uuid = "494afd89-becb-516b-aafa-70d2670c0337"
+repo = "https://github.com/JuliaInterop/JavaCall.jl.git"
+
+[deps]
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
```

Comment:
Since it uses WinReg, and you've added a Project.toml file, you'll need to add it to [deps] as well.
This is why AppVeyor is failing.